lib/redis/namespace.rb

```diff
@@ -366,7 +366,7 @@ def method_missing(command, *args, &block)
 
     def inspect
       "<#{self.class.name} v#{VERSION} with client v#{Redis::VERSION} "\
-      "for #{@redis.id}/#{@namespace}>"
+      "for #{@redis.id}/#{full_prefix}>"
```

Comment:
Can't we use the `full_namespace` method https://github.com/resque/redis-namespace/blob/e722410b73ac7adb5204f85b0cde38bca97a6e4a/lib/redis/namespace.rb#L312?

Reply:
Sure, I just rebased the last changes.
Thanks a lot for your tip.